### ISSUE TITLE
NullPointerException in OrientJdbcStatement.mayCleanForSpark

### ISSUE THREAD

**Issue by eirikur-as** [orientechnologies/orientdb#8831]
## Expected behavior  
No exception

## Actual behavior  
```java
java.lang.NullPointerException
	at com.orientechnologies.orient.jdbc.OrientJdbcStatement.mayCleanForSpark(OrientJdbcStatement.java:363)
	at com.orientechnologies.orient.jdbc.OrientJdbcStatement.execute(OrientJdbcStatement.java:94)
	at com.orientechnologies.orient.jdbc.OrientJdbcStatement.executeQuery(OrientJdbcStatement.java:121)
```

## Steps to reproduce  
```java
package com.test;

import com.orientechnologies.orient.core.db.ODatabaseType;
import com.orientechnologies.orient.core.db.OrientDB;
import com.orientechnologies.orient.core.db.OrientDBConfig;
import com.orientechnologies.orient.jdbc.OrientDataSource;
import org.testng.annotations.Test;

import java.sql.Connection;
import java.sql.SQLException;
import java.sql.Statement;

public class OrientMemDbTest {

    @Test
    public void testJdbcStatement() throws SQLException {
        final String serverUser = "admin";
        final String serverPassword = "admin";
        final String dbName = "test";

        OrientDB orientDB = new OrientDB("embedded:.", serverUser, serverPassword, OrientDBConfig.defaultConfig());
        orientDB.create(dbName, ODatabaseType.MEMORY);

        OrientDataSource ods = new OrientDataSource(orientDB, dbName);
        Connection connection = ods.getConnection(serverUser, serverPassword);
        Statement statement = connection.createStatement();
        statement.executeQuery("SELECT FROM V");
    }
}
```

**Comment by luigidellaquila** [comment]
Hi @eirikur-as 

I'm checking it, I'll let you know ASAP

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @eirikur-as 

I just pushed a fix, the snapshot will be available in a few hours.
It will be released with v 3.0.19

Thanks

Luigi
